src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -438,29 +438,126 @@ public void OnWindowUpdate(int amount)
                 }
             }
 
+            private const int FirstHPackRequestPseudoHeaderId = 1;
+            private const int LastHPackRequestPseudoHeaderId = 7;
+            private const int FirstHPackStatusPseudoHeaderId = 8;
+            private const int LastHPackStatusPseudoHeaderId = 14;
+            private const int FirstHPackNormalHeaderId = 15;
+            private const int LastHPackNormalHeaderId = 61;
+
+            private static readonly int[] HPackStaticStatusCodeTable = new int[LastHPackStatusPseudoHeaderId - FirstHPackStatusPseudoHeaderId + 1] { 200, 204, 206, 304, 400, 404, 500 };
+
+            private static readonly (HeaderDescriptor descriptor, byte[] value)[] HPackStaticHeaderTable = new (HeaderDescriptor, byte[])[LastHPackNormalHeaderId - FirstHPackNormalHeaderId + 1]
+            {
+                (KnownHeaders.AcceptCharset.Descriptor, Array.Empty<byte>()),
+                (KnownHeaders.AcceptEncoding.Descriptor, Encoding.ASCII.GetBytes("gzip, deflate")),
```

Comment:
nit: This data already exists in H2StaticTable - https://github.com/dotnet/runtime/blob/3873bf5c11512433c97d88c0cc1279354d70d682/src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/H2StaticTable.cs#L45

Get the header field from the table and use its copy of `"gzip, deflate"` byte array? Probably a little micro-optimizy 😄

Reply:
Yeah, it's just one value, I'm not that concerned.